benchmarks/torchbench_model.py

```diff
@@ -154,8 +155,37 @@
     "detectron2_fcos_r_50_fpn",
 }
 
+# This list was extracted from PyTorch's repository: benchmarks/dynamo/torchbench.py
 FORCE_FP16_FOR_BF16_MODELS = {"vision_maskrcnn"}
 
+# Some models have large dataset that doesn't fit in memory. Lower the batch
+# size to test the accuracy.
+# This list was extracted from PyTorch's repository: benchmarks/dynamo/torchbench.py
```

Comment:
Do you think we can factor these lists out there and reuse them? That way we would be more robust against divergence.